app/src/main/java/com/github/se/orator/model/theme/AppThemeViewModel.kt

```diff
@@ -4,14 +4,28 @@ import android.content.Context
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.asStateFlow
 
+/** Enum class holding the possible values for the app theme. */
+enum class AppThemeValue {
+  LIGHT,
+  DARK,
+  SYSTEM_DEFAULT
+}
+
 /**
  * ViewModel managing the current theme of the app.
  *
  * @param context The context of the app.
  */
 class AppThemeViewModel(private val context: Context) {
 
-  private val IS_DARK_NAME = "isDark"
```

Comment:
Interesting approach introducing currentTheme as a StateFlow in AppThemeViewModel.